src/components/inspector/data-mapping.vue

```diff
@@ -12,8 +12,8 @@
     <table class="table table-striped table-sm border mb-1">
       <thead>
         <tr>
-          <th scope="col">{{ $t('Source') }}</th>
-          <th scope="col">{{ $t('Form Variable') }}</th>
+          <th scope="col">{{ $t('Source') }} <span class="badge badge-info" v-pre>{{ M }}</span> </th>
+          <th scope="col">{{ $t('Form Variable') }} <span class="badge badge-info" v-pre>{{ M }}</span> </th>
```

Comment:
Can you use src/components/inspector/mustache-helper.vue component for this?

Reply:
Fixed